src/NzbDrone.Core/Download/Clients/DownloadStation/Proxies/DiskStationProxyBase.cs

```diff
@@ -172,40 +112,62 @@ private string GenerateSessionCacheKey(DownloadStationSettings settings)
 
         private string AuthenticateClient(DownloadStationSettings settings)
         {
-            var arguments = new Dictionary<string, object>
-            {
-                 { "api", "SYNO.API.Auth" },
-                 { "version", "1" },
-                 { "method", "login" },
-                 { "account", settings.Username },
-                 { "passwd", settings.Password },
-                 { "format", "sid" },
-                 { "session", "DownloadStation" },
-             };
-
             var authInfo = GetApiInfo(DiskStationApi.Auth, settings);
 
-            var requestBuilder = BuildRequest(settings, authInfo, HttpMethod.GET);
+            var requestBuilder = BuildRequest(settings, authInfo, "login", 2);
```

Comment:
But someone may want to use it as client only (sonarr running on other server). That said, i agree with you that we should use v2.